src/legacy/core_plugins/expressions/public/np_ready/public/render.ts

```diff
@@ -108,7 +111,7 @@ export class ExpressionRenderHandler {
 
     try {
       // Rendering is asynchronous, completed by handlers.done()
-      getRenderersRegistry()
+      await getRenderersRegistry()
```

Comment:
Error was swallowed here. Have to await for promise to capture error in catch clouse